keras/callbacks.py

```diff
@@ -549,6 +549,12 @@ class ProgbarLogger(Callback):
     # Raises
         ValueError: In case of invalid `count_mode`.
     """
+    verbose = None
+    epochs = None
+    log_values = None
+    target = 0
+    seen = 0
+    progbar = None
```

Comment:
Instance attributes should never be class attributes. They should be defined in __init__. You may not realize it, but everything you define at the class level (like here) is *shared across instances*.